webapp/channels/src/components/threading/global_threads/thread_item/thread_item.tsx

```diff
@@ -75,9 +76,10 @@ function ThreadItem({
     threadId,
     isFirstThreadInList,
     isPostPriorityEnabled,
+    routing,
 }: Props & OwnProps): React.ReactElement|null {
     const dispatch = useDispatch();
-    const {select, goToInChannel, currentTeamId} = useThreadRouting();
```

Comment:
We needed to make this component agnostic to routing, so that it can be used in other routes than global threads.